packages/material-ui/src/Collapse/Collapse.js

```diff
@@ -147,7 +179,7 @@ const Collapse = React.forwardRef(function Collapse(props, ref) {
   const handleExited = normalizedTransitionCallback(onExited);
 
   const handleExiting = normalizedTransitionCallback((node) => {
-    const wrapperHeight = wrapperRef.current ? wrapperRef.current.clientHeight : 0;
+    const wrapperSize = wrapperRef.current[isHorizontal ? 'clientWidth' : 'clientHeight'];
```

Comment:
This used to be guarded against wrapperRef.current being falsy. 

It does seem wrong to guard this way but let's keep the guard just in case.